hail/python/hailtop/hailctl/dev/config/cli.py

```diff
@@ -1,21 +1,31 @@
 import os
 import json
+from hailtop.config import get_deploy_config
 
 
 def init_parser(parser):
+    parser.add_argument("namespace", type=str, nargs='?',
```

Comment:
Should this be `--namespace`?

Reply:
I changed this on purpose to be a positional argument rather than one with a flag.  So now you can say `hailctl dev config cseed` to make `cseed` the default namespace.